specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1128 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HDInsightJobManagementClient",
+    "description": "The HDInsight Job Client.",
+    "version": "2018-11-01-preview",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "parameters": [
+      {
+        "$ref": "#/parameters/endpointParameter"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/livy/batches": {
+      "get": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "List all spark batch jobs",
+        "operationId": "SparkBatch_List",
+        "x-ms-examples": {
+          "List all spark batch jobs": {
+            "$ref": "./examples/HDI_SparkBatch_List.json"
+          }
+        },
+        "consumes": [
+          "application/json"
```

Comment:
Your call don't take a body, there should not have a `consumes` node. Please apply the concept "no body input => no consumes" to the entire file.

Reply:
Fixed it. Thanks for your comment.